monkey/monkey_island/cc/services/attack/mitre_api_interface.py

```diff
@@ -0,0 +1,13 @@
+from stix2 import FileSystemSource, Filter
+
+
+class MitreApiInterface:
+
+    ATTACK_DATA_PATH = 'monkey_island/cc/services/attack/attack_data/enterprise-attack'
+
+    @staticmethod
+    def get_all_mitigations() -> list:
+        file_system = FileSystemSource(MitreApiInterface.ATTACK_DATA_PATH)
+        mitigation_filter = [Filter('type', '=', 'course-of-action')]
+        all_mitigations = file_system.query(mitigation_filter)
+        return all_mitigations
```

Comment:
Would be very easy to add a unit test to this - just check that one mitigation we know about is in the list + there are at least X items in the list. You can read about "learning tests" in the Unit Test part of Clean Code to get a sense of what I'm aiming to